test/swiss_schema_test.exs

```diff
@@ -43,6 +45,18 @@ defmodule SwissSchemaTest do
       end
     end
 
+    test "load :repo option from default app config" do
+      Application.put_env(:swiss_schema, SwissSchema, Repo, persistent: false)
+
+      defmodule CustomSchema do
+        use Ecto.Schema
+        use SwissSchema
+      end
+
+      assert function_exported?(CustomSchema, :aggregate, 1)
+      assert function_exported?(CustomSchema, :aggregate, 2)
+    end
+
```

Comment:
We should test if compilation fails when config is missing (both inline `:repo` and app env `:default_repo` is absent).

TBH, I have no idea how to do it 😂 but I'll check the docs when I have a chance. I'll report back here if I find smt interesting.